src/main/java/com/google/cloud/tools/action/app/config/DefaultDeployConfiguration.java

```diff
@@ -32,16 +31,16 @@
   private final Collection<Path> deployables;
   private final String bucket;
   private final String dockerBuild;
-  private final Boolean force;
+  private final boolean force;
```

Comment:
So these values will always have to be provided? What's the rationale behind this? We're going to have to assume default values in our lib, instead of just using gcloud's ones.